File: client/makc2021-react-redux-ts/src/Layer5/Pages/DummyMain/List/Queries/Get/DummyMainListPageGetQueryOutput.ts
// Copyright (c) 2021 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

import {
  createDomainListGetQueryOutput,
  DomainListGetQueryOutput
} from 'src/Layer4/Domains/DummyMain/Queries/List/Get/DomainListGetQueryOutput';

/**
 * Выходные данные запроса на получение страницы сущностей "DummyMain".
 */
export interface DummyMainListPageGetQueryOutput {
  /**
   * Список.
   */
  list: DomainListGetQueryOutput;
}

/**
 * Создать выходные данные запроса на получение страницы сущностей "DummyMain"
 * @returns Выходные данные запроса на получение страницы сущностей "DummyMain".
 */
export function createDummyMainListPageGetQueryOutput() {
  return {
    list: createDomainListGetQueryOutput()
  } as DummyMainListPageGetQueryOutput;
}

File: server/src/Makc2021.Layer5.Sql.Server/Pages/DummyMain/List/Queries/Get/DummyMainListPageGetQueryOutput.cs
﻿// Copyright (c) 2021 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

using Makc2021.Layer4.Sql.Domains.DummyMain.Queries.List.Get;

namespace Makc2021.Layer5.Sql.Server.Pages.DummyMain.List.Queries.Get
{
    /// <summary>
    /// Выходные данные запроса на получение страницы сущностей "DummyMain".
    /// </summary>
    public class DummyMainListPageGetQueryOutput
    {
        #region Properties

        /// <summary>
        /// Список.
        /// </summary>
        public DomainListGetQueryOutput List { get; set; }

        #endregion Properties
    }
}
